ElectronClient/gui/NoteEditor/NoteBody/CodeMirror/Editor.tsx

```diff
@@ -85,6 +89,7 @@ function Editor(props: EditorProps, ref: any) {
 	useScrollUtils(CodeMirror);
 	useCursorUtils(CodeMirror);
 	useLineSorting(CodeMirror);
+	useEditorSearch(CodeMirror);
 	useJoplinMode(CodeMirror);
```

Comment:
I hadn't noticed this but it's good how you've added additional behaviour to the editor. Just custom hooks in separate files with only CodeMirror as a dependency - that makes it very easy to see what's going on and to maintain it 👍

Reply:
Thanks, I was following similar convention from what you had done with ace :)